app/notes/[noteId].tsx

```diff
@@ -0,0 +1,74 @@
+import { useFocusEffect, useLocalSearchParams } from 'expo-router';
+import { useCallback, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { colors } from '../../constants/colors';
+import type { NoteResponseBodyGet } from '../api/[noteId]+api';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: colors.background,
+    padding: 10,
+  },
+  textContainer: {
+    alignItems: 'center',
+    gap: 12,
+  },
+  title: {
+    fontSize: 24,
+    fontFamily: 'Poppins_700Bold',
+    color: colors.text,
+    textAlign: 'center',
+  },
+  textContent: {
+    fontSize: 16,
+    fontFamily: 'Poppins_400Regular',
+    color: colors.textSecondary,
+    textAlign: 'center',
+    marginTop: 8,
+  },
+});
+
+export default function NotePage() {
```

Comment:
I guess this component name should match the filename?

Reply:
The single guest component is also named `GuestPage`, or in our Next.js example we use `AnimalPage`

https://github.com/upleveled/next-js-example-fall-2024-atvie/blob/27d3d175cab732e60ce34549fa507ab3d98da1ca/app/animals/%5BanimalId%5D/page.tsx#L24

Reply:
Yeah, here we'll diverge from Next.js, because it's not a page

Reply:
0893801b68d42f913d4ce291d138358d712b4577